application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -30,8 +34,11 @@
     private final List<Command> commandList = new ArrayList<>();
     private final Map<String, Command> commandMap;
 
-    public CommandHandler() {
-        commandList.addAll(List.of(new ReloadCommand(this)));
+    public CommandHandler(Database database) {
+        TagSystem tagSystem = TagSystemInstanceHolder.getOrCreate(database);
```

Comment:
I would prefer it if we could add the whole tag command family with a single command to keep the changes to this file minimal.

`CommandHandler` should ideally just add the command family in one argument and thats it.

Would it be easy to refactor it in a way to keep changes to this file as minimal as possible?

Reply:
I don't understand what you mean with this, where else am I supposed to register the commands?